### PR TITLE
add smoke test for uperf workload

### PR DESCRIPTION
**Pull request by mohit-sheth**
helps with testing the changes to the scale-ci-pipeline

### CODE REVIEW

File: workloads/network-perf/smoke_test.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      protos:
+        - tcp
+      test_types:
+        - stream
+      sizes:
+        - 1024
+      runtime: 30
+EOF
+
+sleep 30
```
Comment (chaitanyaenr) [inline workloads/network-perf/smoke_test.sh:69]: What's the need for this sleep?

Comment (mohit-sheth) [reply workloads/network-perf/smoke_test.sh:69]: it takes some time for the benchmark operator to come up

File: workloads/network-perf/smoke_test.sh [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+fi
+
+if [[ ${BASELINE_POD_2P_UUID} ]]; then
+  _baseline_pod_2p_uuid=${BASELINE_POD_2P_UUID}
+fi
+
+if [[ ${BASELINE_POD_4P_UUID} ]]; then
+  _baseline_pod_4p_uuid=${BASELINE_POD_4P_UUID}
+fi
+
```
Comment (jtaleric) [inline workloads/network-perf/smoke_test.sh:23]: Seems like all the above could be removed? Since we won't be comparing in the smoke test, right?

Comment (mohit-sheth) [reply workloads/network-perf/smoke_test.sh:23]: so the smoke test's primary reason is to test the PRs from scale-ci-pipeline. 
Pipeline basically git clones and runs either hn2hn, pod or service which will take 1 hour each at best. The parameters inside like runtime, message sizes, protos are not configurable from the pipeline perspective.
Hence this smoke test will be a quick way to do e2e tests for PR

Comment (jtaleric) [reply workloads/network-perf/smoke_test.sh:23]: ack

**Review by jtaleric — APPROVED** [review]
LGTM

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**